src/schnorr.js

```diff
@@ -15,6 +15,7 @@ const zero = BigInteger.ZERO;
 function sign(privateKey, message, aux) {
   // https://github.com/bitcoin/bips/blob/master/bip-0340.mediawiki#signing
   check.checkSignParams(privateKey, message);
+  privateKey = typeof (privateKey) == 'string' ? BigInteger.fromHex(privateKey) : privateKey;
```

Comment:
We should also update the pseudo API doc in the README to reflect this change.

Reply:
👍

Reply:
Actually, I meant this: https://github.com/guggero/bip-schnorr/blob/master/README.md#schnorrsignprivatekey--biginteger-message--buffer--buffer
But that's okay, I can update that after merging.